src/components/PreviewComment/index.js

```diff
@@ -143,12 +147,20 @@ export const styles = StyleSheet.create({
   seemore: {
     color: COLORS.blue
   },
-  profileTouchable: {marginLeft: -12},
   image: {
     width: 24,
     height: 24,
     borderRadius: 12,
     justifyContent: 'center',
     alignItems: 'center'
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Consider moving these inline styles into the StyleSheet at the bottom of the file.

```diff
- image: {
-   width: 24,
-   height: 24,
-   borderRadius: 12,
-   justifyContent: 'center',
-   alignItems: 'center'
- },
+ image: styles.image,
```
In your StyleSheet:
```javascript
image: {
  width: 24,
  height: 24,
  borderRadius: 12,
  justifyContent: 'center',
  alignItems: 'center'
}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->